st3/sublime_lib/glob_util.py

```diff
@@ -0,0 +1,21 @@
+import re
+from functools import lru_cache
+
+
+GLOB_RE = re.compile(r'(\*\*/?|\*)')
+
+
+@lru_cache()
+def get_glob_expr(pattern):
+    s = ''
+    for part in GLOB_RE.split(pattern):
+        if part == '**/':
+            s += r'(?:.*/)?'
+        elif part == '**':
+            s += r'(?:.*)'
+        elif part == '*':
+            s += r'(?:[^/]*)'
+        else:
```

Comment:
What about `?` characters? They should probably be converted to `.`.

I suppose we can ignore `[set]` matching.